app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -278,12 +275,17 @@
 // The select component dropdown could be anchored to the page body.
 // In this case we moved the dropdown styles out of the `.select-component` class.
 .selectize-dropdown {
+  // variable that did not exists in tabler but used by selectize.
+  --bs-border-color-translucent: var(--tblr-border-color-translucent);
```

Comment:
moved variable here because it uses for dropdown, and dropdown could be anchored to the body
it fixes problem with disappeared border In team grid